_posts/2018-09-14-Episode-2-XND.md

```diff
@@ -0,0 +1,74 @@
+Welcome to Open Source Directions, I am your host Anthony Scopatz, and joining me is Travis Oliphant and Saul Shanabrook.  First Travis will introduce himself.
+
+I am Travis Oliphant, founder and board member of Anaconda, founder and advisory board member at Numfocus, and currently CEO of Quansight.  I have been involved in a number of open source projects, over the past 20 years and am passionate about them.
+
+Now Saul will introduce himself.
+
+I am Saul Shanabrook, I have been working at Quansight for just a little while now with XND and I am happy to be here.
+
+Now we will move on to the headline section and go around sharing recent events.  First Saul:
+
+There is a project called Learna which is a mono-repo management tool for JavaScript, and is actually something we use in Jupyter Lab.  Earlier last week, they changed their license from MIT to MIT minus people who work with ICE and businesses that work with ICE.  This change did not last, but it happened for about a day or so.  The basic idea here is that this instance illustrates the ability for developer communities to exert authority over how their tools are used.  I am interested to see where this discussion will go, and the larger implications with ethics.
+
+(Travis) I am excited by the recent Numfocus announcements about their new board selection process and various efforts which show the maturity of the organization.  They have 5 great new board members and it is great to see them join Andy and Murena who are continuing with all the work they achieve behind the scenes.  In addition to the board, I am impressed by all the work on the part of staff and volunteers and love the great work they do to make PyData and everything work in that organization.
```

Comment:
In an effort to make this more narrative, please remove the names here like, Travis, Saul, Anthony, etc.

Reply:
Would that be all instances of names, or did you just mean the parenthetical notations?

Reply:
Any of the parentehtical names that just refer to who was speaking.  Who was speaking is not super interesting for the blog